Updating base.html

Leeadar opened the pull request:
- Updating navigation bar
- Adding register and login pages to the navbar
- After user logs in, the user sees profile and logout pages
- If the user try to reach http://127.0.0.1:8000/profile/ before logging in, the user redirect to login page. 
The URL will be http://127.0.0.1:8000/login/?next=/profile/

closes #57

Yarboa commented on pickATrip_django_apps/static/pickATrip_django_apps/main.css:
```diff
@@ -0,0 +1,84 @@
+
+.account-heading {
+  font-size: 2.5rem;
+}
```
Please insert new line
Avoid CSS, please continue with bootstrap
make the base.html visible 1st

Yarboa commented on pickATrip_django_apps/templates/pickATrip_django_apps/base.html:
```diff
@@ -1,62 +1,73 @@
-	<div id ="content" class = "container">
-		{% block content %}{% endblock %}
-	</div>
+    <link rel="stylesheet" type="text/css" href="{% static 'pickATrip_django_apps/main.css' %}">
```
Please avoid CSS

Leeadar committed:
Updating base.html

- Adding register and login pages
- After login the user see profile and logout pages

Leeadar commented:
@Yarboa I deleted the CSS file and updated base.html

Yarboa reviewed (approved):
Looks good to me

guy9050 reviewed (approved):
lgtm
